wander-embedded-sdk/src/components/iframe/wander-iframe.component.ts

```diff
@@ -233,6 +233,7 @@ export class WanderIframe {
     }
 
     this.wrapper.dataset.layout = layoutType;
+    this.backdrop.dataset.layout = layoutType;
```

Comment:
I think the only change needed is to swap these lines in `wander-iframe.component.ts`:

    shadow.appendChild(backdrop);
    shadow.appendChild(halfImage);
    shadow.appendChild(wrapper);

For:

    shadow.appendChild(wrapper);
    shadow.appendChild(halfImage);
    shadow.appendChild(backdrop);

Just make sure z-indexes are properly set.

This way, they can target the backdrop for a specific layout type like so:

   .wrapper[data-layout="popup"] + .backdrop